src/components/dialogs/network-modifications/by-formula/formula/formula-form.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { elementType, SelectInput } from '@gridsuite/commons-ui';
+import {
+    EDITED_FIELD,
+    EQUIPMENT_TYPE_FIELD,
+    FILTERS,
+    OPERATOR,
+    REFERENCE_FIELD_OR_VALUE_1,
+    REFERENCE_FIELD_OR_VALUE_2,
+} from '../../../../utils/field-constants';
+import { useWatch } from 'react-hook-form';
+import DirectoryItemsInput from '../../../../utils/rhf-inputs/directory-items-input';
+import { gridItem } from '../../../dialogUtils';
+import { EQUIPMENTS_FIELDS } from './formula-utils';
+import ReferenceAutocompleteInput from './reference-autocomplete-input';
+import DragHandleIcon from '@mui/icons-material/DragHandle';
+
+interface FormulaProps {
+    name: String;
+    index: number;
+}
+
+export const OPERATOR_OPTIONS = [
+    { id: 'ADDITION', label: '+' },
```

Comment:
You don't need to put the operations in the label; we don't need to translate them